packages/rancher-monitoring/generated-changes/patch/Chart.yaml.patch

```diff
@@ -11,6 +11,8 @@
 +  catalog.cattle.io/provides-gvr: monitoring.coreos.com.prometheus/v1
 +  catalog.cattle.io/display-name: "Monitoring"
 +  catalog.cattle.io/auto-install: rancher-monitoring-crd=match
++  catalog.cattle.io/requests-cpu: "4500m"
++  catalog.cattle.io/requests-memory: "4000Mi"
```

Comment:
Breakdown for why these values were chosen is found in https://github.com/rancher/rancher/issues/30684#issuecomment-765648833